aliyun-ecs/src/test/java/org/jclouds/aliyun/ecs/compute/ECSComputeServiceLiveTest.java

```diff
@@ -54,6 +46,7 @@ protected Properties setupProperties() {
       Properties properties = super.setupProperties();
       vpcId = setIfTestSystemPropertyPresent(properties,  provider + ".vpcId");
       vSwitchId = setIfTestSystemPropertyPresent(properties,  provider + ".vSwitchId");
+      group = "jclouds"; // otherwise jclouds will use provider name as group but `aliyun` is a forbidden prefix for tags
```

Comment:
Then we should probably preconfigure the `prefix` property in the provider metadata too, so users don't have to do it manually. We must provide defaults that work.

Reply:
good point, Ill look into it

Reply:
@nacx as it can't start with `aliyun` shall I use `alibaba-ecs` as id for the provider in the `ECSComputeServiceProviderMetadata` ?

Reply:
Sounds good. Preconfigure that and remove this from here.